src/components/Checkbox/index.tsx

```diff
@@ -21,9 +21,9 @@ interface TooltipOffsets {
 }
 
 const tooltipOffsets: TooltipOffsets = {
-    default: [-11, -1],
-    top: [-22, -3],
-    shared: [-8, 8],
+    default: [0, -3],
```

Comment:
Yes

Reply:
I ask because it seems a bit unusual that they changed this much in relation to an ant-d update; but the check-boxes do appear to be in the right place after the update